modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java

```diff
@@ -72,6 +73,8 @@ public Map<String, Object> getParameters(
 			GetterUtil.getBoolean(ddmFormField.getProperty("alphabeticalOrder"))
 		).put(
 			"dataSourceType", ddmFormField.getDataSourceType()
+		).put(
+			"defaultSearch", _hasDefaultSearch(ddmFormField)
```

Comment:
Hey @p-albuquerque,
We could replace this method with this statement:
`GetterUtil.getBoolean(ddmFormField.getProperty("defaultSearch"))` 

What do you think?

Reply:
i will see if this works, thanks